src/main/java/duke/list/List.java

```diff
@@ -0,0 +1,92 @@
+package duke.list;
+
+import duke.DukeException;
+
+public class List {
+
+    private static Task[] items;
+    private int size;
+
+    public List(){
+        items = new Task[100];
```

Comment:
Maybe you can try using ArrayList from java's packages so that you won't have to declare a fixed size array. What happens if user inputs more than 100 tasks?